intel_extension_for_transformers/neural_chat/pipeline/plugins/audio/tts.py

```diff
@@ -76,9 +76,9 @@ def __init__(self, output_audio_path="./response.wav", voice="default", stream_m
             self.pat_speaker_embeddings = torch.load(os.path.join(asset_path, 'speaker_embeddings/spk_embed_pat.pt'))
 
         self.cpu_pool = None
-        if not torch.cuda.is_available():
+        if self.device == 'cpu':
             # ipex IOMP hardware resources
-            if self.device == 'cpu' and 'LD_PRELOAD' in os.environ and 'libiomp' in os.environ['LD_PRELOAD']:
+            if 'LD_PRELOAD' in os.environ and 'libiomp' in os.environ['LD_PRELOAD']:
```

Comment:
We can remove such env checkers in the following PR.

Reply:
Understood, I will remove this.